src/pynwb/file.py

```diff
@@ -491,11 +492,17 @@ def __init__(self, **kwargs):
         icephys_electrodes = args_to_set['icephys_electrodes']
         ic_electrodes = args_to_set['ic_electrodes']
         if icephys_electrodes is None and ic_electrodes is not None:
```

Comment:
```suggestion
        if ic_electrodes is not None:
```
I think we should raise the error regardless of whether `icephys_electrodes` is also provided.